routes/index.js

```diff
@@ -2,8 +2,8 @@ var express = require('express');
 var router = express.Router();
 
 /* GET home page. */
-router.get('/', function(req, res, next) {
-  res.render('index', { title: 'Express' });
+router.get('/dashboard', function(req, res, next) {
```

Comment:
This is confusing. Now when I'm authenticated and go to "/", I get a 404, since there is no handler anymore for "/"